### ISSUE TITLE
coreos.filesystem.writabledirs is flaky

### ISSUE THREAD

**Issue by mischief** [coreos/mantle#377]
```
find: `/etc/gshadow.lock': No such file or directory
find: `/etc/shadow.lock': No such file or directory
find: `/etc/passwd.lock': No such file or directory
find: `/etc/group.lock': No such file or directory
...
2016-08-23T03:27:09Z kola: --- FAIL: coreos.filesystem.writabledirs on gce (34.599s)
2016-08-23T03:27:09Z kola:         Failed to run find: output [], status: Process exited with: 1. Reason was:  ()
```

it looks like there's a race of find reading the direntries of /etc. possibly racing against the gce agent adding users.

what's a sane way to fix this race? stop `google-accounts-manager.service`?

**Comment by mischief** [comment]
actually any test in `kola/tests/misc/files.go` is probably affected.

**Comment by dm0-** [comment]
Fixed in #430.
